core/config/locales/en.yml

```diff
@@ -1644,6 +1644,7 @@ en:
         available_locales: This determines which locales are available for your customers to choose from in the storefront.
         cart_tax_country_iso: 'This determines which country is used for taxes on carts (orders which don''t yet have an address).<br> Default: None.'
         code: An identifier for your store. Developers may need this value if you operate multiple storefronts.
+        default_currency: This determines which currency will be used to retrieve product prices for this store. If you want to change it globally, set `Spree::Config.currency` instead. <br><br>Please, be aware that only products that have prices in the selected currency will be listed.
```

Comment:
I'm still a bit confused. What do we mean by "retrieve"? Do we mean from the frontend (although we should be agnostic)? Is it also true from the API? Do we mean the default product searcher? On top of that, I think (not 100% sure), that `Spree::Config.currency` is the only one used when creating products on the backend.

Reply:
It's used by the `current_pricing_options` helper, which is used by all frontends, API and base searcher, so I think it's safe to assume that it's a valid general rule for any "storefront" instance (in fact, it is available in the store preference). 

It seems to be a different thing for the backend, and probably `Spree::Config.currency` is used to set the default currency because the same product can be available in different stores.

If you have any suggestions on how to make it clearer, please help me! 🙂

Reply:
Rephrased, what about now?

Reply:
Cool!